pkg/download/downloader.go

```diff
@@ -72,6 +75,18 @@ func extractZIP(targetDir string, read io.ReaderAt, size int64) error {
 			return errors.Wrap(err, "could not open inflating zip file")
 		}
 
+		if f.FileInfo().Mode()&os.ModeSymlink == os.ModeSymlink {
+			buf, err := ioutil.ReadAll(src)
+			if err != nil {
+				return errors.Wrap(err, "problem reading symlink target")
+			}
+			oldname := string(buf)
```

Comment:
var name "oldname" doesn't make sense there, something like dst or target is better

Reply:
oldname comes from the documentation for os.Symlink: https://golang.org/pkg/os/#Symlink